040_More_Documentation.html.md

```diff
@@ -13,5 +13,33 @@ This section describes the details of your solution in a logical order. For exam
 
 #### Example
 For an example of unique solution documentation, see the <a href="https://devportal.yaas.io/solutions/saphybrisprofile/index.html">SAP Hybris Profile</a> documentation.
+
+### README files
+
+README files describe other files in a given directory.
```

Comment:
I would combine these, and make more active:
README files describe other files in a given directory. Use this template to create external README files: